classes/class-pmpro-members-list-table.php

```diff
@@ -36,6 +36,21 @@ public function __construct() {
 				// If true, the parent class will call the _js_vars() method in the footer
 			)
 		);
+
+		// Check for default hidden columns.
+		$this->get_hidden_columns();
```

Comment:
Is this line necessary? `get_hidden_columns()` doesn't look like it does anything besides for return the array of hidden columns, which we don't use here.

Reply:
Technically it would save the user meta if there was nothing there already too. But the method gets called during the prepare_items() method as well, which seems enough to set the defaults on fresh loads (no user meta) and to save/check/uncheck items.

I'm removing it. Nice catch.